src/SWP/Bundle/ContentBundle/Rule/Applicator/ArticleRuleApplicator.php

```diff
@@ -77,6 +88,10 @@ public function apply(RuleInterface $rule, RuleSubjectInterface $subject)
 
         $subject->setTemplateName($configuration[$this->supportedKeys[1]]);
 
+        if ((bool) $configuration[$this->supportedKeys[2]]) {
```

Comment:
I don't like relying on array index. Why not `if (in_array('published', $configuration) && true === $configuration['published'])`